### PR TITLE
Logo update and docs cleanup

### PR DESCRIPTION
**Pull request by pepov**
- remove obsolete docs
- update logo

### CODE REVIEW

File: docs/Readme.md [hunk 1]
```diff
@@ -1 +0,0 @@
-The documentation of the Logging operator project is available at the [Banzai Cloud Documentation Page](https://banzaicloud.com/docs/one-eye/logging-operator/).
```
Comment (OverOrion) [inline docs/Readme.md:1]: [optional]
I am fine with deleting it from there as the new docs are mentioned in the root README already, just wanted to ask why not change it to the newer one?

Comment (pepov) [reply docs/Readme.md:1]: I'm going to provide a short description on what kind of docs are here instead of linking to the site

**Comment by pepov** [comment]
waiting for a logo update, as the colored logo is not entirely visible on a white background

**Review by ahma — APPROVED** [review]
lgtm

**Pull request merged**